src/main/java/org/candlepin/subscriptions/event/EventController.java

```diff
@@ -69,6 +69,7 @@ public class EventController {
   private final OptInController optInController;
   private final TransactionHandler transactionHandler;
   private final EventConflictResolver eventConflictResolver;
+  private static final String AAP_CLUSTER = "AAP Cluster";
```

Comment:
nitpick: move the static fields above at the same place than EXCLUDE_LOG_FOR_EVENT_SOURCES

Reply:
Done